packages/contracts/package.json

```diff
@@ -5,7 +5,9 @@
   "private": true,
   "scripts": {
     "----- EXTERNAL -----": "---------------------------",
+    "init": "yarn tablegen && yarn build && yarn worldgen && yarn worldtypes",
     "prepare": "yarn tablegen && yarn worldgen && yarn worldtypes",
```

Comment:
this would get run twice now, right? once on the repo root's `prepare` calling `init` and then again when calling this package's `prepare`?

Reply:
I don't think the package's `prepare` runs by default though, or does it?

Reply:
got it! I assumed `prepare` ran as part of `install`